driver/health/event_manager_default_test.go

```diff
@@ -0,0 +1,49 @@
+package health
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	rulereadiness "github.com/ory/oathkeeper/rule/readiness"
+)
+
+func TestNewDefaultHealthEventManager(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	t.Run("health event manager", func(t *testing.T) {
+		ruleReadinessProbe := rulereadiness.NewReadinessHealthChecker()
+		hem := NewDefaultHealthEventManager()
+
+		if err := hem.AddListener(ruleReadinessProbe); err != nil {
+			t.Errorf("AddListener() error = %v", err)
+			return
+		}
```

Comment:
We use https://github.com/stretchr/testify usually for assertions:
```suggestion
		require.NoError(t, hem.AddListener(ruleReadinessProbe))
```